SVProfilePictureViewer/PictureViewerForm.cs

```diff
@@ -121,6 +126,7 @@ enum MaskType {
           g.DrawImage(dark, 0, 0);
           return light;
         }
+        //light still a bit off, but it's the best to see on pc screen
```

Comment:
When you say "light still la bit is off" do  you mean the lighting in the resulting image is a bit off or something else?